airflow-core/src/airflow/models/dag.py

```diff
@@ -236,6 +236,7 @@ def _create_orm_dagrun(
     creating_job_id: int | None,
     backfill_id: NonNegativeInt | None,
     triggered_by: DagRunTriggeredByType,
+    triggering_user: str | None = None,
```

Comment:
```suggestion
    triggering_user_name: str | None = None,
```

Because here again I expect a User object by reading this.

Reply:
Good point. Will adjust as proposed to make it clear.